src/lbfgs.jl

```diff
@@ -188,13 +188,11 @@ function solve!(
     end
   end
 
-  return GenericExecutionStats(
-    status,
-    nlp,
-    solution = x,
-    objective = f,
-    dual_feas = ∇fNorm,
-    iter = iter,
-    elapsed_time = elapsed_time,
-  )
+  stats = GenericExecutionStats(status, nlp)
```

Comment:
Should the constructor `GenericExecutionStats(status, nlp, kwargs...)` be deprecated now? Unless, we plan to modify it to set the `reliable` booleans.

Reply:
I introduced a simplified constructor in https://github.com/JuliaSmoothOptimizers/SolverCore.jl/pull/31.

However, the current constructor remains necessary for those who want special storage for the bounds multipliers (e.g., sparse vectors).

Reply:
@tmigot On @geoffroyleconte's recommendation, I removed the "simplified" constructor as it was simply redundant.

Reply:
Awesome, thanks!